packages/web/docs/src/components/oss-friends.tsx

```diff
@@ -19,22 +19,14 @@ export async function getStaticProps() {
     }>;
   } = await res.json();
 
-  return {
-    props: {
-      ssg: { friends: body.data },
-    },
-  };
+  return body.data;
 }
 
-export function OSSFriendsPage(props: {
-  friends: Array<{
-    name: string;
-    description: string;
-    href: string;
-  }>;
-}) {
+export async function OSSFriendsPage() {
+  const friends = await fetchFriends();
+
   return (
-    <Page className="text-green-1000 light mx-auto max-w-[90rem] overflow-hidden">
+    <LandingPageContainer className="text-green-1000 light mx-auto max-w-[90rem] overflow-hidden">
       <div className="bg-beige-100 relative isolate mx-4 flex max-w-[90rem] flex-col gap-6 overflow-hidden rounded-3xl px-4 py-6 max-sm:mt-2 sm:py-12 md:mx-6 md:gap-8 lg:py-24">
```

Comment:
```suggestion
      <div className="bg-beige-100 relative isolate mx-4 flex flex-col gap-6 overflow-hidden rounded-3xl px-4 py-6 max-sm:mt-2 sm:py-12 md:mx-6 md:gap-8 lg:py-24">
```
since you added to parent